libact/query_strategies/tests/test_realdata.py

```diff
@@ -0,0 +1,112 @@
+"""
+This module uses real world dataset to test the active learning algorithms.
+Since creating own artificial dataset for test is too time comsuming so we would
+use real world data, fix the random seed, and compare the query sequence each
+active learning algorithm produces.
+"""
```

Comment:
I'll change its description a bit just leave it as example how the future developers can do to make the test more rigorous. This file shouldn't affect user too much.